### ISSUE TITLE
Tooltip moves tab focus/order to bottom of the page when using keyboard

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#6664]
This was originally reported over in Slack, the tooltip seems to be causing focus to jump to the end of the page:

![demo](https://user-images.githubusercontent.com/3901764/90183747-a5c2a680-dd79-11ea-8d89-a4fc200c04a3.gif)

Instead, the tooltip should be in focus order and leaving the tooltip should go to the next item in the tab sequence.

**Comment by joshblack** [comment]
Going to close this as it seems like it was addressed in the latest release!
